<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true"/>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="room">
           <property name="font">
            <font>
             <family>Poor Richard</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CUSTOM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="leaveButton">
              <property name="font">
               <font>
                <family>Poor Richard</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius : 100; 
border : 2px solid black;
padding-bottom: 5px</string>
              </property>
              <property name="text">
               <string>Leave room</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QWidget" name="widget_14" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="widget_19" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="ingame1">
                 <property name="font">
                  <font>
                   <family>Poor Richard</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Host</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Poor Richard</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../resources/white_pawn.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_15" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Poor Richard</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>vs</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="playButton">
                 <property name="font">
                  <font>
                   <family>Poor Richard</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>OpenHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-radius : 100; 
border : 2px solid black;
padding-bottom: 5px</string>
                 </property>
                 <property name="text">
                  <string>play</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_16" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="ingame2">
                 <property name="font">
                  <font>
                   <family>Poor Richard</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Guest</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>Poor Richard</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../resources/black_pawn.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_17" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget_22" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_13"/>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="inviteGroup" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_14"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="messageScreen" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1088</width>
              <height>335</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2"/>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_12" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="inputMessage">
              <property name="font">
               <font>
                <family>Poor Richard</family>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendMessage">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Poor Richard</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius : 100; 
border : 2px solid black;
padding-bottom: 5px</string>
              </property>
              <property name="text">
               <string>send</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
